auth/src/main/java/com/redhat/lightblue/rest/authz/RolesCache.java

```diff
@@ -48,19 +48,7 @@ public void onRemoval(@ParametersAreNonnullByDefault RemovalNotification notific
         fallbackRolesCache = CacheBuilder.newBuilder()
                 .concurrencyLevel(10) // handle 10 concurrent request without a problem
                 .maximumSize(500) // Hold 500 sessions before remove them
-                .expireAfterWrite(5, TimeUnit.HOURS) // If the session is inactive for more than 5 hours, remove it
-                .removalListener(
-                        new RemovalListener<String, List<String>>() {
-                            {
-                                LOGGER.debug("Removal Listener created");
-                            }
-
-                            @Override
-                            public void onRemoval(@ParametersAreNonnullByDefault RemovalNotification notification) {
-                                LOGGER.debug("This data from " + notification.getKey() + " evacuated due:" + notification.getCause());
-                            }
-                        }
-                ).build();
+                .build();
```

Comment:
Every time roles are fetch successfully from ldap they're updated in both caches:

https://github.com/lightblue-platform/lightblue-rest/blob/master/auth/src/main/java/com/redhat/lightblue/rest/authz/RolesCache.java#L67-L71

So yes the fallback will not expire but it will kept updated.  If a client is idle for a period of time and then ldap search fails, would it not be better to use what we knew in the past?  The cache has a size limit so it can't grow beyond 500, but that's the only limit.